Automattic/woocommerce-payments
# Add payment methods logos

Fixes #298 

#### Changes proposed in this Pull Request

* Add payment method brand image to transaction list and payment method details

#### Testing instructions

1. Make sure all tests are passing
2. Open up the transaction list, you should see the brand showing up for charge, refund and adjustment transactions
![image](https://user-images.githubusercontent.com/7714042/70538224-81566080-1b40-11ea-8a32-69288fec9623.png)
3. Click on the transaction details button, you should see the brand under payment methods
![image](https://user-images.githubusercontent.com/7714042/70538235-84515100-1b40-11ea-805d-e1835c5cb911.png)

-------------------
